### PR TITLE
accessability fixes for detailed package

### PR DESCRIPTION
**Pull request by Sreenivas-Kalluru**
this fixes accessibility issues for the below pages

- https://vcpkg.io/en/package/boost-core.html
- https://vcpkg.io/en/package/boost-static-assert.html
- https://vcpkg.io/en/package/boost-config.html
- https://vcpkg.io/en/package/boost-integer.html
- https://vcpkg.io/en/package/boost-container-hash.html
- https://vcpkg.io/en/package/boost-coroutine.html
- https://vcpkg.io/en/package/boost-assert.html
- https://vcpkg.io/en/package/boost-atomic.html
- https://vcpkg.io/en/package/boost-iterator.html
- https://vcpkg.io/en/package/boost-concept-check.html
- https://vcpkg.io/en/package/boost-mpl.html
- https://vcpkg.io/en/package/boost-type-traits.html
- https://vcpkg.io/en/package/boost-tuple.html
- https://vcpkg.io/en/package/boost-parameter.html
- https://vcpkg.io/en/package/boost-smart-ptr.html
- https://vcpkg.io/en/package/boost-predef.html
- https://vcpkg.io/en/package/boost-utility.html
- https://vcpkg.io/en/package/boost-align.html
- https://vcpkg.io/en/package/boost-throw-exception.html
- https://vcpkg.io/en/package/boost-array.html
- https://vcpkg.io/en/package/boost-preprocessor.html
- https://vcpkg.io/en/package/boost-move.html
- https://vcpkg.io/en/package/boost-function.html
- https://vcpkg.io/en/package/boost-algorithm.html
- https://vcpkg.io/en/package/boost-winapi.html
- https://vcpkg.io/en/package/boost-system.html
- https://vcpkg.io/en/package/boost-io.html
- https://vcpkg.io/en/package/boost-mp11.html
- https://vcpkg.io/en/package/boost-optional.html
- https://vcpkg.io/en/package/boost-graph.html
- https://vcpkg.io/en/package/boost-describe.html
- https://vcpkg.io/en/package/boost-geometry.html
- https://vcpkg.io/en/package/boost-fusion.html
- https://vcpkg.io/en/package/boost-detail.html
- https://vcpkg.io/en/package/boost-functional.html
- https://vcpkg.io/en/package/boost-function-types.html
- https://vcpkg.io/en/package/boost-exception.html
- https://vcpkg.io/en/package/boost-context.html

### CODE REVIEW

**Comment by Sreenivas-Kalluru** [comment]
@ras0219-msft  @JavierMatosD  Could you please review and let me know if modifications. required

File: css/package.css [hunk 1]
```diff
@@ -359,3 +359,6 @@ ul.tab-container {
     border-radius: 4px;
     width: 20%;
 }
+a {
+    color: #006ee5 !important;
```
Comment (JavierMatosD) [inline css/package.css:363]: I believe the accessibility issue was that the color of version>= was too similar to the dependency name. If so why this change?

Comment (Sreenivas-Kalluru) [reply css/package.css:363]: Hi @JavierMatosD ,

This is the accessibility fix for home page URL highlighted in the picture.
image

**Review by GO-NFT-GO — COMMENTED** [review]
Ok

**Review by GO-NFT-GO — COMMENTED** [review]
Ok

**Review by GO-NFT-GO — COMMENTED** [review]
Ok

**Review by GO-NFT-GO — COMMENTED** [review]
Oi

File: css/package.css (hunk 1 above)
Comment (GO-NFT-GO) [reply css/package.css:363]: Okk

File: css/package.css [hunk 2]
```diff
@@ -359,3 +359,6 @@ ul.tab-container {
...
+  }
```
Comment (GO-NFT-GO) [inline css/package.css:364]: Ok

**Review by GO-NFT-GO — COMMENTED** [review]
Ok

**Review by GO-NFT-GO — COMMENTED** [review]
Ok

**Review by GO-NFT-GO — APPROVED** [review]
Pk

**Review by GO-NFT-GO — COMMENTED** [review]
Ok

**Comment by GO-NFT-GO** [comment]
Ok

**Review by JavierMatosD — APPROVED** [review]

**Review by GO-NFT-GO — COMMENTED** [review]
Ok try
